src/command/mod.rs

```diff
@@ -1,14 +1,14 @@
 mod config;
-mod core;
 mod docs;
 mod graph;
 mod info;
 mod install;
 mod output;
 mod subgraph;
+mod supergraph;
 mod update;
 
-pub use self::core::Core;
+pub use self::supergraph::Supergraph;
```

Comment:
don't need `self` here anymore if it's not named `core`
```suggestion
pub use supergraph::Supergraph;
```